app/models/conversion_host/configurations.rb

```diff
@@ -41,14 +41,22 @@ def enable(params)
       params = params.symbolize_keys
       _log.info("Enabling a conversion_host with parameters: #{params}")
 
-      params.delete(:task_id)     # In case this is being called through *_queue which will stick in a :task_id
-      params.delete(:miq_task_id) # The miq_queue.activate_miq_task will stick in a :miq_task_id
+      task_id = params.delete(:task_id)         # In case this is being called through *_queue which will stick in a :task_id
+      miq_task_id = params.delete(:miq_task_id) # The miq_queue.activate_miq_task will stick in a :miq_task_id
+      task_id ||= miq_task_id
```

Comment:
What? no no lets just fix enable_queue to set `:miq_task_id` and standardize on a single key

Reply:
Yup. I didn't like it neither. How would you fix `enable_queue` ?

Reply:
Actually it looks like the only caller (./manageiq-api/app/controllers/api/conversion_hosts_controller.rb) just does `task_id = ConversionHost.enable_queue(data.except('auth_user'), data['auth_user'])` so where were you seeing a `task_id` parameter coming from?

Reply:
I took that from existing code... @djberg96 could you shed some light here ?

Reply:
Yes, the `enable_queue` method will return the `task_id` on a create request, but we don't have a direct way to associate the task back to the conversion instance after that.

Reply:
But enable_queue isn't passing in the task_id as a parameter right?  So the only option should be `miq_task_id`

Reply:
They're injected into the params via the `MiqTask.generic_action_with_callback` at https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_task.rb#L290-L292

Reply:
But those are different hashes, only `task_id` is passed in to the method opts